tmd/io/io.py

```diff
@@ -162,16 +162,25 @@ def load_population(neurons, user_tree_types=None, name=None, use_morphio=False)
     elif os.path.isdir(neurons):  # Assumes given input is a directory
         files = [os.path.join(neurons, neuron_dir) for neuron_dir in os.listdir(neurons)]
         name = name if name is not None else os.path.basename(neurons)
+    elif os.path.isfile(neurons):  # Assumes given input is a file
+        files = [neurons]
+        name = name if name is not None else os.path.basename(neurons)
+    else:
+        raise TypeError(
+            f'The type of the given neurons ({type(neurons)}) is not supported or the path does '
```

Comment:
Showing the type was not a great suggestion from my part.  How about sth like:
```python
f'The format of the given neurons is unknown. '
'Expected an iterable of files, or a directory, or a single morphology file. '
f'Got: {neurons}'
```

Reply:
Even better indeed, thanks